src/Ovp/Util/OvpUtil.php

```diff
@@ -0,0 +1,52 @@
+<?php
+declare(strict_types=1);
+
+namespace Triniti\Ovp\Util;
+
+use Triniti\Ovp\ArtifactUrlProvider;
+use Triniti\Schemas\Dam\AssetId;
+use Gdbots\Pbj\WellKnown\NodeRef;
+use Gdbots\Pbj\Message;
+
+
+final readonly class OvpUtil {
+    public function __construct(private readonly ArtifactUrlProvider $artifactUrlProvider)
+    {
+    }
+
+    public function getUrl($id, string $type): ?string
```

Comment:
@digitalartcrew this method name was ok before when it was only intended to be called via the twig function, but as part of a util it's not great

think about how it would be calling it in another class:

`$url = $ovpUtil->getUrl($id, 'manifest');`

it no longer tells you much about what kind of url is returned